.gitignore

```diff
@@ -1,5 +1,7 @@
 src/qinfer/version.py
 
+*.sublime-workspace
+
```

Comment:
I always reject them. If the configuration files hold value, I'd rather see them as examples in some sort of IDE-config folder in the documentation (with the prefix-dot removed). Its a huge pain and I'd be swearing every time I opened the project if I introduced changes to anything under the IDE-config folder.